.github/labeler.yml

```diff
@@ -40,3 +40,21 @@
 "C:CLI":
   - client/**/*
   - x/*/client/**/*
+# Add "C:docs" label to documented related files and directories. 
+"C:docs":
+  - x/epochs/specs/*
+  - x/gamm/specs/*
+  - x/incentives/specs/*
+  - x/lockup/specs/*
+  - x/mint/specs/*
+  - x/pool-incentives/specs/*
+  - x/simulation/specs/*
+  - x/superfluid/specs/*
+  - x/tokenfactory/specs/*
+  - x/txfees/specs/*
+  - x/README.md
+  - README.md
```

Comment:
Would it match README's in other folders such as `tests/e2e/README.md`? If not, I'm wondering if it is possible to reach any README with something like:
```
**/README.md
```

Reply:
Agreed, I think it'd be best to change to `**/README.md`?

Reply:
Ok I simplified the rules, github does not allow **/README.md. or **/**/*.md  or **/*/*.md 
I made it:

```
"C:docs":
  - CHANGELOG.md
  - bug.md
  - issue.md
  - proto-docs.md
  - README.md
  - x/**/*.md
  - x/**/*/*.md
  - tests/**/*.md
  
```